<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Repositories</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="repositoryListWidget"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="addRepositoryLineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="addRepositoryButton">
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeRepositoryButton">
             <property name="text">
              <string>Remove selected</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Select profile to load:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QComboBox" name="repositoryProfilesComboBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="saveProfileButton">
             <property name="text">
              <string>Save new profile</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="loadProfileButton">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="deleteProfileButton">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>181</width>
             <height>249</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="getRepositoryListButton">
           <property name="text">
            <string>Get repository list from internet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Build system settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>CPU Arch:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cpuArchComboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>i686</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>i486</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x86_64</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Compiled package output:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QLineEdit" name="pkgOutputEdit">
           <property name="text">
            <string>/var/mpkg/packages/</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QComboBox" name="cpuTuneComboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>i686</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>i486</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>x86_64</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>CPU Tune:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Source cache directory:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" rowspan="3">
          <widget class="QLineEdit" name="sourceCacheEdit">
           <property name="text">
            <string>/var/mpkg/source_cache/</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Job count:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Maintainer name:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="maintainerNameEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Optimization level:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Maintainer e-mail:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="maintainerEmailEdit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Output package type:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="outputPkgTypeComboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>txz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tgz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tlz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tbz</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="oLevelComboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentIndex">
            <number>2</number>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>O0</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>O1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>O2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>O3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Os</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="numjobsSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>802</width>
           <height>198</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MPKG core settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="disableDependenciesCheckBox">
         <property name="text">
          <string>Disable dependencies</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>CD-ROM device:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="cdromDeviceEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="enableDownloadResumeCheckBox">
         <property name="toolTip">
          <string>By default, download resume is enabled for wget and aria2 tools. If you want to specify exact behaviour, choose some of this.</string>
         </property>
         <property name="text">
          <string>Enable download resume</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>CD-ROM mount point:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="cdromMountEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="enablePrelinkCheckBox">
         <property name="text">
          <string>Enable prelink</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="enablePrelinkRandomCheckBox">
         <property name="text">
          <string>Enable stack randomization in prelink</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="enableBDeltaCheckBox">
         <property name="toolTip">
          <string>Enable support for delta updates. This feature extremely saves traffic, but eats your HDD and CPU. Support for this feature was dropped in repositories and may be removed permanently from mpkg.</string>
         </property>
         <property name="text">
          <string>Enable bdelta support (deprecated)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Download tool:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QComboBox" name="downloadToolComboBox">
         <item>
          <property name="text">
           <string>wget</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>aria2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>mpkg (old default)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>By default, mpkg now uses wget as the most robust tool. If you want more speed, choose aria2 or mpkg.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Update blacklist</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>These packages will not be updated automatically. List of such packages stored in /etc/mpkg-update-blacklist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="updateBlacklistListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="addUpdateBlacklistLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="addUpdateBlacklistButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeUpdateBlacklistButton">
           <property name="text">
            <string>Remove selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Remove protected</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>List of packages protected from removal. You will need to enter special key to enable removing any of them.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="removeProtectedListWidget"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="addRemoveProtectedLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="addRemoveProtectedButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeRemoveProtectedButton">
           <property name="text">
            <string>Remove selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Unremovable packages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>List of packges, which files will be never removed from it's locations (even if package will be purged). It is dangerous to edit this, so it is read-only here. To edit manually, see /etc/mpkg-unremovable</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="unremovableListWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>repositoryListWidget</tabstop>
  <tabstop>addRepositoryLineEdit</tabstop>
  <tabstop>addRepositoryButton</tabstop>
  <tabstop>removeRepositoryButton</tabstop>
  <tabstop>repositoryProfilesComboBox</tabstop>
  <tabstop>saveProfileButton</tabstop>
  <tabstop>loadProfileButton</tabstop>
  <tabstop>deleteProfileButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>cpuArchComboBox</tabstop>
  <tabstop>cpuTuneComboBox</tabstop>
  <tabstop>numjobsSpinBox</tabstop>
  <tabstop>oLevelComboBox</tabstop>
  <tabstop>outputPkgTypeComboBox</tabstop>
  <tabstop>pkgOutputEdit</tabstop>
  <tabstop>sourceCacheEdit</tabstop>
  <tabstop>maintainerNameEdit</tabstop>
  <tabstop>maintainerEmailEdit</tabstop>
  <tabstop>disableDependenciesCheckBox</tabstop>
  <tabstop>enableDownloadResumeCheckBox</tabstop>
  <tabstop>enablePrelinkCheckBox</tabstop>
  <tabstop>enablePrelinkRandomCheckBox</tabstop>
  <tabstop>enableBDeltaCheckBox</tabstop>
  <tabstop>cdromDeviceEdit</tabstop>
  <tabstop>cdromMountEdit</tabstop>
  <tabstop>updateBlacklistListWidget</tabstop>
  <tabstop>addUpdateBlacklistLineEdit</tabstop>
  <tabstop>addUpdateBlacklistButton</tabstop>
  <tabstop>removeUpdateBlacklistButton</tabstop>
  <tabstop>removeProtectedListWidget</tabstop>
  <tabstop>addRemoveProtectedLineEdit</tabstop>
  <tabstop>addRemoveProtectedButton</tabstop>
  <tabstop>removeRemoveProtectedButton</tabstop>
  <tabstop>unremovableListWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
